### DISCUSSION TITLE
[自定义主页] Vocaloid 随机 Wiki!

### DISCUSSION THREAD

**Discussion by LinQingYuu** [Meloong-Git/PCL#5209] [自定义主页]
因为...
服务端其实还没写好，自动化脚本也还没搓
所以这是一个前瞻体验版本（
暂时没考虑进预设
## 订阅地址

https://raw.githubusercontent.com/LTY-Followers/VocaloidWikiPageForPCL2/refs/heads/main/resolved/%E5%85%89%E4%B8%8E%E5%BD%B1%E7%9A%84%E5%AF%B9%E7%99%BD.xaml

因为只搓好了光与影（但是有些没时间做）

## 预览效果

![捕获](https://github.com/user-attachments/assets/6cb19f8e-fa74-42e0-a4a8-1d8ffb2c0fb7)

## 我觉得目前问题比较大的地方

![2](https://github.com/user-attachments/assets/3ecaed82-9e91-486f-bb3c-65496a66ac9f)

截图可能有点糊，可以在 PCL 上预览

欢迎各位提出建议和改进方案！

**Comment by Joker2184** [comment]
![QQ截图20241203172310](https://github.com/user-attachments/assets/ad42895b-00c1-49d6-a454-2d991fa4117e)
![8004F499FF3F4D7F13DFFC601F8A36B1](https://github.com/user-attachments/assets/9243e0e9-2d68-41ae-b9c3-a654215a14f0)
 你Button怎么能卡出来的

**Reply by LinQingYuu** [reply]
>你Button怎么能卡出来的

在你预览之前我就发现了这个问题，并且推了个更新上去，但是因为 GitHub Raw 缓存刷新需要几分钟时间... 所以你看到的是已经过时的页面（

**Reply by Joker2184** [reply]
6666

**Comment by Joker2184** [comment]
然后东西有点杂（ 需要重新整理下UI（（（

**Reply by LinQingYuu** [reply]
> 然后东西有点杂（ 需要重新整理下UI（（（

文本可能有点乱，因为我真不大熟 WPF
结构是按萌娘百科的条目的，其实还有个注释没加的说（

Edit:[萌娘百科页面](https://mzh.moegirl.org.cn/%E5%85%89%E4%B8%8E%E5%BD%B1%E7%9A%84%E5%AF%B9%E7%99%BD)

**Comment by LTCatt** [comment]
如果考虑进预设了可以艾特我……

**Reply by LinQingYuu** [reply]
>如果考虑进预设了可以艾特我……

ok

**Comment by LinQingYuu** [comment]
## 检查项（等差不多勾完了应该就能上了）

- [x] 自动化数据处理
- [x] bilibili 图片获取
- [x] 萌娘百科内容搜索和获取
- [ ] 主页构建器
- [x] 图片自动转码和上传图床
- [ ] VCPedia 数据搜索和获取
- [ ] 服务端

Edit:仓库现已迁移至组织账号以方便管理
